src/parse/asp/interpreter.go

```diff
@@ -164,7 +164,9 @@ func (i *interpreter) getConfig(state *core.BuildState) *pyConfig {
 
 	i.configMutex.Lock()
 	defer i.configMutex.Unlock()
-	i.config[state.Config] = c
+	if state.FinishedPreloading {
```

Comment:
If we cache this before we finish pre-loading, we'd end up caching the state without the plugin config.